app/src/androidTest/java/org/mozilla/focus/activity/DownloadFileTest.kt

```diff
@@ -104,7 +108,9 @@ class DownloadFileTest {
             clickDownloadButton()
             verifySnackBarText("finished")
             mDevice.openNotification()
-            verifyDownloadNotification()
+            notificationTray {
+                expandEraseBrowsingNotification()
```

Comment:
I think this should be verifySystemNotificationExists, as the expandEraseBrowsingNotification does something else.